dotnet/Capstone/Controllers/BirdNoteController.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Capstone.DAO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System;
+using Capstone.Models;
+using Capstone.Exceptions;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdNoteController : ControllerBase
+    {
+        private readonly IBirdNoteDao birdSightingDao;
+        public BirdNoteController(IBirdNoteDao birdSightingDao)
+        {
+            this.birdSightingDao = birdSightingDao;
+
+        }
+
+        [HttpGet("/bird/{birdId}/notes")]
+        public List<BirdNote> getNotes(int birdId)
+        {
+            List<BirdNote> sightingList = birdSightingDao.getSightings(birdId);
+
+            if (sightingList == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sightingList;
+
+        }
+
+        [HttpGet("/note")]
+        public BirdNote getNote(int id)
```

Comment:
Should this limit access to notes owned by user?

Reply:
Actually, looking at the BirdNoteService.js, there's no id being passed. That said, it's unclear what the intention of this method was, and there are no uses of this route in the supplied front end.

Reply:
I can't see a use for it as well especially without an id being passed. I will still leave it and maybe we will find a its use later.